src/System.IO.FileSystem/tests/Directory/Regressions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Regressions : FileSystemTest
```

Comment:
I don't think we need to call these out as regression tests.  In general we add lots of tests to fill gaps where we previously had test gaps, and how we discover such gaps really doesn't matter.  In a year's time it won't really matter whether we added this test because we spotted it as a regression or not.  I would just add this to whatever existing test class is most closely aligned to what's being tested, e.g. to our test class for enumerators.

Reply:
Ok, will change.